app/components/primer/navigation/tab_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render Primer::BaseComponent.new(**@system_arguments) do %>
+  <%= icon %>
+  <%= title %>
+  <%= counter %>
```

Comment:
I would have expected to see `panel` in here somewhere. How does this work?  ;)

Reply:
the panel position is implemented by the parent, because according to `https://github.com/github/tab-container-element` the panels are siblings to the `nav`
https://github.com/primer/view_components/blob/e0548ac0cc09a982c2858596b548e43ed5f9e459/app/components/primer/tab_nav_component.html.erb#L2-L14